static/app/views/insights/types.tsx

```diff
@@ -15,7 +15,7 @@ export enum ModuleName {
   RESOURCE = 'resource',
   AI = 'ai',
   MOBILE_UI = 'mobile-ui',
-  MOBILE_SCREENS = 'mobile-screens',
+  MOBILE_SCREENS = 'mobile-vitals',
```

Comment:
Should we also rename the `MOBILE_SCREENS` enum to `MOBILE_VITALS` for consistent terminology?

Reply:
Good point, it's done!